### PR TITLE
feat(TBD-12824): cleanup Pivotal on components

### PR DESCRIPTION
**Pull request by sponomarova**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.libraries.bonita/.project [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lbourgeois) [inline main/plugins/org.talend.libraries.bonita/.project:1]: Is this change intended ?

File: main/plugins/org.talend.libraries.jersey/.project [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lbourgeois) [inline main/plugins/org.talend.libraries.jersey/.project:1]: Is this change intended ?

File: main/plugins/org.talend.libraries.saaj/.project [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lbourgeois) [inline main/plugins/org.talend.libraries.saaj/.project:1]: Is this change intended ?

File: test/plugins/org.talend.designer.scd.test/targets/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/org/
```
Comment (lbourgeois) [inline test/plugins/org.talend.designer.scd.test/targets/.gitignore:1]: Is this change intended ?

File: --form [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN""http://www.w3.org/TR/html4/strict.dtd">
```
Comment (lbourgeois) [inline --form:1]: Wrong commit I guess
Same for 2 other empty files :
--header
--url

**Review by lbourgeois — CHANGES_REQUESTED** [review]

**Review by lbourgeois — APPROVED** [review]

**Review by firefryy — APPROVED** [review]
lgtm

**Pull request merged**